pwa/src/templates/templateParts/dependenciesTemplates/networkDependenciesTemplate/NetworkDependenciesTemplate.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from "react";
+import * as styles from "./NetworkDependenciesTemplate.module.css";
+import { Network } from "vis-network";
+import { useTranslation } from "react-i18next";
+import _ from "lodash";
+import { getTokenValue } from "../../../../services/getTokenValue";
+import { addNewLineToString } from "../../../../services/addNewLineToString";
+
+interface LayersResultTemplateProps {
+  components: any[];
+  mainComponent: {
+    id: string;
+    name: string;
+    layer: string;
+  };
+}
+
+export const NetworkDependenciesTemplate: React.FC<LayersResultTemplateProps> = ({ components, mainComponent }) => {
+  const { t } = useTranslation();
+
+  const container = React.useRef(null);
+
+  const componentNodes = components.map((component) => ({
+    id: component.id,
+    label: addNewLineToString(component.name),
+    layer: component.embedded?.nl?.embedded?.commonground?.layerType,
+    color: {
+      background: getTokenValue(
+        styles[_.camelCase(`layerColor ${t(_.upperFirst(component.embedded?.nl?.embedded?.commonground?.layerType))}`)],
+      ),
+      border: getTokenValue(
+        styles[
+          _.camelCase(`borderLayerColor ${t(_.upperFirst(component.embedded?.nl?.embedded?.commonground?.layerType))}`)
+        ],
+      ),
+      highlight: {
+        background: getTokenValue(
+          styles[
+            _.camelCase(
+              `highlightLayerColor ${t(_.upperFirst(component.embedded?.nl?.embedded?.commonground?.layerType))}`,
+            )
+          ],
+        ),
+        border: getTokenValue(
+          styles[
+            _.camelCase(`layerColor ${t(_.upperFirst(component.embedded?.nl?.embedded?.commonground?.layerType))}`)
+          ],
+        ),
+      },
+    },
```

Comment:
I would like to completely remove the highlight state and borders. To maintain simplicity in the frontend design _and_ to reduce complexity here.